Open Judge System/Web/OJS.Web/Views/Account/Login.cshtml

```diff
@@ -16,7 +16,7 @@
 <div class="row">
     <div class="col-md-8">
         <section id="loginForm">
-            @using (Html.BeginForm("Login", "Account", new { ViewBag.ReturnUrl }, FormMethod.Post, new { @class = "form-horizontal", role = "form" }))
+            @using (Html.BeginForm("Login", "Account", new { ViewBag.ReturnUrl }, FormMethod.Post, new { @class = "form-horizontal", role = "form", id = "formId" }))
```

Comment:
No need to add this id. You can get the form by using the id of section, which is "loginForm"

Reply:
It can be done as:
`var loginFormSection = document.getElementById("loginForm");`
`var loginForm = loginFormSection.getElementsByTagName("form")[0];`
`loginForm.action = loginForm.action + window.location.hash;`
However, getting the first element that matches a form tag in the section seems hacky, so I opted out of it, as relying on the form to be the first/only form element in the section did not seem like a good practice. Is it a preferable solution to the problem?